app/src/main/java/org/oppia/app/utility/ClickableAreasImage.kt

```diff
@@ -59,18 +60,25 @@ class ClickableAreasImage(
 
   /** Get X co-ordinate scaled according to image.*/
   private fun getXCoordinate(x: Float): Float {
-    val rect = attacher.displayRect
     return (x * rect.width()) + rect.left
   }
 
   /** Get Y co-ordinate scaled according to image.*/
   private fun getYCoordinate(y: Float): Float {
-    val rect = attacher.displayRect
     return (y * rect.height()) + rect.top
   }
 
+  private fun getImageViewWidth(): Int {
```

Comment:
@BenHenning should i add kdoc here ?
It's quite straight forward so that's why didn't add one

Reply:
I don't think it's needed, but suggest maybe using ``getImageViewContentWidth`` and ditto for height to be a bit more specific. I think the computation is straightforward, but it can be a bit more self-documenting.